.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+| Q                 | A
+| ----------------- | ---
+| Bug fix?          | yes/no
+| Test needed?      | yes/no
+| Issues progressed | elifesciences/jira-import#..
+
+<!--
+Write a short README entry for your feature/bugfix here (replace this comment block.)
```

Comment:
you don't strictly _need_ to replace the comment block as it won't be visible; but you're welcome to do it